src/main/java/jalau/at18/katas/pokerhands/daniela/CardValue.java

```diff
@@ -1,27 +1,37 @@
 package jalau.at18.katas.pokerhands.daniela;
 
 public enum CardValue {
-    ACE(null),
-    KING(ACE),
-    QUEEN(KING),
-    JACK(QUEEN),
-    TEN(JACK),
-    NINE(TEN),
-    EIGHT(NINE),
-    SEVEN(EIGHT),
-    SIX(SEVEN),
-    FIVE(SIX),
-    FOUR(FIVE),
-    THREE(FOUR),
-    TWO(THREE);
+    ACE(null, 14),
+    KING(ACE, 13),
+    QUEEN(KING, 12),
+    JACK(QUEEN, 11),
+    TEN(JACK, 10),
+    NINE(TEN, 9),
+    EIGHT(NINE, 8),
+    SEVEN(EIGHT, 7),
+    SIX(SEVEN, 6),
+    FIVE(SIX, 5),
+    FOUR(FIVE, 4),
+    THREE(FOUR, 3),
+    TWO(THREE, 2);
 
     private CardValue next;
+    private int valuecard;
 
-    CardValue(CardValue next) {
+    CardValue(CardValue next, int value) {
         this.next = next;
+        this.valuecard = value;
     }
 
     public CardValue next() {
         return next;
     }
+
+    public int getRealNumberValue() {
```

Comment:
This should be removed if it is not needed anymore

Reply:
Already removed